english/Canada/Canada.md

```diff
@@ -0,0 +1,14 @@
+Well, looks like the end of a long and beautiful empire. (Insert name of politician you don't like) has won. What do you do?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.